### ISSUE TITLE
用語集：スタイルプロパティの見直し

### ISSUE THREAD

**Issue by momdo** [waic/wcag21#1767]
related: #98

https://waic.jp/translations/WCAG21/#dfn-style-properties
> スタイルプロパティ (style properties)
ユーザエージェントによってコンテンツ要素が (画面、音声スピーカー、点字ディスプレイなどを介して) レンダリングされる際の提示のされ方 (フォント、色、サイズ、位置、パディング、音量、合成音声の韻律など) を定義するプロパティ。
スタイルプロパティには、オリジンがいくつか考えられる。
ユーザエージェントのデフォルトのスタイル: コンテンツ制作者や利用者がスタイルを指定しない場合は、ユーザエージェントのデフォルトのスタイルプロパティが使用される。ウェブコンテンツ技術には、デフォルトの提示方法を指定しているものとしていないものがある。
コンテンツ制作者のスタイル: コンテンツ制作者がコンテンツの一部として設定するスタイルプロパティ (インラインスタイル、コンテンツ制作者のスタイルシートなど)。
利用者によるスタイル: 利用者が設定するスタイルプロパティ (ユーザエージェントのインタフェース設定や利用者によるスタイルシートなど)。

https://www.w3.org/TR/2018/REC-WCAG21-20180605/#dfn-style-properties
> property whose value determines the presentation (e.g. font, color, size, location, padding, volume, synthesized speech prosody) of content elements as they are rendered (e.g. onscreen, via loudspeaker, via braille display) by user agents
Style properties can have several origins:
User agent default styles: The default style property values applied in the absence of any author or user styles. Some web content technologies specify a default rendering, others do not;
Author styles: Style property values that are set by the author as part of the content (e.g. in-line styles, author style sheets);
User styles: Style property values that are set by the user (e.g. via user agent interface settings, user style sheets)

案：
ユーザエージェントによってコンテンツ要素が (画面、音声スピーカー、点字ディスプレイなどを介して) レンダリングされるとき、その値の提示 (フォント、色、サイズ、位置、パディング、音量、合成音声の韻律など) を定義するプロパティ。
スタイルプロパティには、オリジンがいくつか考えられる。
ユーザエージェントのデフォルトのスタイル: コンテンツ制作者又は利用者のスタイルがない場合は、ユーザエージェントのデフォルトのスタイルプロパティが適用される。ウェブコンテンツ技術には、デフォルトのレンダリングを指定するものと、指定しないものがある。
コンテンツ制作者のスタイル: コンテンツ制作者がコンテンツの一部として設定するスタイルプロパティの値 (インラインスタイル、コンテンツ制作者のスタイルシートなど)。
利用者によるスタイル: 利用者が設定するスタイルプロパティの値 (ユーザエージェントのインタフェース設定、利用者のスタイルシートなど)。

**Comment by bakera** [comment]
user style が「利用者のスタイル」「利用者によるスタイル」で揺れているので、揃えた方が良さそうです。「利用者のスタイル」に統一が良いでしょうか。


>ウェブコンテンツ技術には、デフォルトのレンダリングを指定するものと、指定しないものがある。
>Some web content technologies specify a default rendering, others do not;

この「指定する」はちょっとわかりにくいように思いました (コンテンツ制作者が指定する、というようにも読める)。ここの specify は「仕様によってが決められている」というようなニュアンスだと思いますので、「規定している」「規定されている」のような感じの方が良いかもしれません (他の場所でspecifyってどうしてましたっけ……?)

- ウェブコンテンツ技術には、デフォルトのレンダリングを規定しているものと、規定していないものがある。
- ウェブコンテンツ技術には、デフォルトのレンダリングが規定されているものと、されていないものがある。


user style sheetsを「利用者のスタイルシート」としていますが、1.3.1解説書では「ユーザスタイルシート」としているようです。

https://waic.jp/translations/WCAG21/Understanding/info-and-relationships.html
>例えば、コンテンツがスクリーンリーダーによって読み上げられたり、コンテンツ制作者が提供するスタイルシートがユーザスタイルシートに置き換えられたりしたときには、提示形式が変わる。

https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html
> For example, the presentation format changes when the content is read by a screen reader or when a user style sheet is substituted for the style sheet provided by the author.

他の箇所でどうしているのかはすぐにはわからず……どちらが優勢なのか調べて揃える感じでしょうか。

**Comment by momdo** [comment]
> 他の箇所でどうしているのかはすぐにはわからず……どちらが優勢なのか調べて揃える感じでしょうか。

概ねユーザスタイルシートに統一しているみたいですね。利用者のスタイルシートというのは他ではみあたらず。
```
./techniques/aria/ARIA24.html:            <p>一部のロービジョンの利用者は、テキストコンテンツを認識するためにユーザスタイルシート、スクリプト、又は拡張機能を使いページ上のフォントを上書きする必要がある。しかし、お気に入りを示す星やリンク内のメールアイコンなど、意味のあるアイコンフォントは認識できる必要がある。
./techniques/css/C23.html:            <p>コンテンツ制作者がメインコンテンツ部分の文字色と背景色を指定しなければ、ユーザスタイルシートを使うことなしに、ブラウザでそれらの色を変更することが可能になる。主要コンテンツ以外に対して文字色と背景色を指定するということは、ブラウザはそれらの色を変更しないことを意味する。
./techniques/css/C35.html:            <p>この達成方法の目的は、利用者がユーザスタイルシート、ブックマークレット、拡張機能、又はアプリケーションを介してテキスト間隔を上書きできるようにすることである。段落、行、単語、及び文字の間隔を広げることは、ロービジョン又は認知障害のある人に役立つ。コンテンツは、テキストを含む要素を必要に応じて拡大できるようにすることで、コンテンツ又は機能を損なうことなく間隔を変更できる必要がある。
./techniques/css/C36.html:            <p>この達成方法の目的は、利用者がユーザスタイルシート、ブックマークレット、拡張機能、又はアプリケーションを介して、テキストの間隔を上書きできるようにすることである。段落、行、単語、及び文字の間隔を広げることで、ロービジョン又はいくらかの認知障害のある人にとってメリットとなる。コンテンツは、コンテンツ又は機能を損なうことなく、間隔の変更を可能にする必要があるため、テキストコンテナは、テキストを拡張できるスペースがあるか、又はコンテナが拡張できるかのいずれかでなければならない。この達成方法は、通常、ナビゲーションバーなどの短い文字列に適用される。長い文字列では、スタイルが変更されたときに、読みやすさのために折り返しが求められる可能性が高くなるためである。
./techniques/failures/F104.html:                  <li>ユーザスタイルシート、ブックマークレット、拡張機能、又はアプリケーションを用いて、達成基準に記載された値で、そのページの CSS を上書きする。
./techniques/general/G156.html:            <p>Firefox 及び Chrome では、例えば Stylus という拡張機能を追加することで、ユーザスタイルシートにより好みの前景色と背景色を指定できる。</p>
./understanding/info-and-relationships.html:            <p>この達成基準の意図は、視覚的又は聴覚的な体裁によって暗に伝えられている情報及び関係性を、その提示形式が変わったときにも保つようにすることである。例えば、コンテンツがスクリーンリーダーによって読み上げられたり、コンテンツ制作者が提供するスタイルシートがユーザスタイルシートに置き換えられたりしたときには、提示形式が変わる。</p>
./understanding/text-spacing.html:                     <li>ユーザーのスタイル: ユーザーが設定するスタイルプロパティ (ユーザエージェントのインタフェース設定やユーザーのスタイルシートなど)。</li>
```

**Comment by momdo** [comment]
いただいたコメントを反映するとこうなりますか。

ユーザエージェントによってコンテンツ要素が (画面、音声スピーカー、点字ディスプレイなどを介して) レンダリングされるとき、その値の提示 (フォント、色、サイズ、位置、パディング、音量、合成音声の韻律など) を定義するプロパティ。
スタイルプロパティには、オリジンがいくつか考えられる。
ユーザエージェントのデフォルトのスタイル: コンテンツ制作者又は利用者のスタイルがない場合は、ユーザエージェントのデフォルトのスタイルプロパティが適用される。ウェブコンテンツ技術には、デフォルトのレンダリングを規定しているものと、規定していないものがある。
コンテンツ制作者のスタイル: コンテンツ制作者がコンテンツの一部として設定するスタイルプロパティの値 (インラインスタイル、コンテンツ制作者のスタイルシートなど)。
利用者のスタイル: 利用者が設定するスタイルプロパティの値 (ユーザエージェントのインタフェース設定、ユーザスタイルシートなど)。

**Comment by bakera** [comment]
良さそうに思います!
